pkg/client/kubernetes/resource.go

```diff
@@ -38,6 +43,7 @@ type ResourceInterface interface {
 	Delete(resource meta.ResourceObject) error
 	UpdateStatus(resource meta.ResourceObject) error
 	Update(resource meta.ResourceObject) error
+	CreateFromYamlContent(yamlContent, namespace string, resourceRef meta.ResourceObject, beforeCreate func(object interface{})) error
```

Comment:
Could you please add the documentation in the interface instead?

Reply:
You mean just right here ?